packages/preset-create-react-app/src/helpers/getReactScriptsPath.ts

```diff
@@ -34,9 +34,15 @@ export const getReactScriptsPath = (): string => {
      * This won't work for Windows users, unless within WSL.
      */
     try {
-      const resolvedBinPath = realpathSync(scriptsBinPath);
-      const scriptsPath = join(resolvedBinPath, '..', '..');
-      return scriptsPath;
+      const scriptsBinPathStat = lstatSync(scriptsBinPath);
+      if (scriptsBinPathStat.isSymbolicLink() === true) {
+        const resolvedBinPath = realpathSync(scriptsBinPath);
+        const scriptsPath = join(resolvedBinPath, '..', '..');
+        return scriptsPath;
+      } else if (scriptsBinPathStat.isFile() === true) {
+        const scriptsPath = join(cwd, '/node_modules/react-scripts');
```

Comment:
Looks like this might need to be changed to use `path.join` for cross-platform-compatibility.